settings/authentication.mdx

```diff
@@ -51,6 +51,36 @@ Mintlify supports single sign-on to your dashboard via SAML and OIDC. If you use
     </Tab>
 </Tabs>
 
+## Google Workspace
+
+<Tabs>
+    <Tab title="SAML">
+        <Steps>
+            <Step title="Create an application">
+                Under `Web and mobile apps`, select `Add custom SAML app` from the `Add app` dropdown.                
+            </Step>
+            <Step title="Send us your IdP information">
+                Copy the provided SSO URL, Entity ID, and x509 certificate and send it to the Mintlify team.
```

Comment:
Are there any instructions on where they could perhaps find the information for Google?